rawx/filerepo.go

```diff
@@ -459,7 +460,8 @@ func setOrHasXattr(path, key, value string) error {
 	if bytes.Equal([]byte(value), buf[:sz]) {
 		return nil
 	}
-	return errors.New("XATTR mismatch")
+	return fmt.Errorf("XATTR '%s' of the '%s' mismatch with '%s'",
```

Comment:
- `mismatches`
- and having something like `XATTR 'user.server.type` of the `/mnt/nvme0n1/OPENIO/rawx-0` mismatches with 'rawx'` is not very "english" I would say

Maybe `XATTR '%s' of '%s' mismatches with '%s'` or `XATTR('%s', '%s') mismatches with '%s'`